server/build.gradle.kts

```diff
@@ -187,21 +160,14 @@ tasks {
                     it.readText().trim()
                 }
 
-                if (zipRevision == webUIRevisionTag)
+                if (zipRevision == webUIRevisionTag) {
                     shouldOverwrite = false
+                }
             }
 
             return shouldOverwrite
         }
 
         overwrite(shouldOverwrite())
     }
-
-    withType<LintTask> {
-        source(files("src/kotlin"))
-    }
-
-    withType<FormatTask> {
-        source(files("src/kotlin"))
-    }
```

Comment:
eh, dose it still work without config?

Reply:
It was moved to the top-level build.gradle.kts